From 746c00de64255fbb9293a88cc7b96cd4e1cf0397 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Wed, 01 Apr 2020 14:01:44 +0300
Subject: [PATCH] [SW-11180] offsets: rename ppboot/preboot offsets

Rename the Ppboot and Preboot SPI flash offsets to legacy.
This helps to differentiate between the old boot way with hard coded
pointers and the new one which is going to be introduced soon.
The new way will involve using the image descriptors instead of hard
coded offsets.

Change-Id: I510aaafc25c39ddda9d8de49f2d8612680214e89
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/app/ppboot.c b/app/ppboot.c
index b78dde2..5961977 100644
--- a/app/ppboot.c
+++ b/app/ppboot.c
@@ -42,7 +42,7 @@
 	}
 
 	*offset = next_image_base_offset(next_img_off, name) +
-							HL_SPI_PREBOOT_OFF;
+						HL_SPI_LEGACY_PREBOOT_OFF;
 
 	return image_pending;
 }
diff --git a/app/preboot.c b/app/preboot.c
index e29a3d0..ff4410c 100644
--- a/app/preboot.c
+++ b/app/preboot.c
@@ -82,7 +82,7 @@
 	char *name = "U-Boot";
 	bool upgrade_pending = false;
 	u32 img_offset = next_image_offset(&upgrade_pending);
-	u32 uboot_offset = img_offset + HL_SPI_UBOOT_OFF;
+	u32 uboot_offset = img_offset + HL_SPI_LEGACY_UBOOT_OFF;
 	char *uboot_image = upgrade_pending ? "candidate" : "stable";
 
 	preboot_puts_raw("Preparing to run U-Boot ");
diff --git a/gaudi/common.h b/gaudi/common.h
index f49d70f..2ea200b 100644
--- a/gaudi/common.h
+++ b/gaudi/common.h
@@ -7,10 +7,10 @@
 
 #include <preboot.h>
 
-#define HL_SPI_NEXT_IMAGE_OFF	0x60000
-#define HL_SPI_MODIFI_IMAGE_OFF	0x60004
-#define HL_SPI_PREBOOT_OFF	0xA0000
-#define HL_SPI_UBOOT_OFF	0xB0000
+#define HL_SPI_NEXT_IMAGE_OFF		0x60000
+#define HL_SPI_MODIFI_IMAGE_OFF		0x60004
+#define HL_SPI_LEGACY_PREBOOT_OFF	0xA0000
+#define HL_SPI_LEGACY_UBOOT_OFF		0xB0000
 
 #define PPBOOT_TEXT_BASE		0x7FF0000000
 #define PREBOOT_TEXT_BASE		(PPBOOT_TEXT_BASE + PPBOOT_SIZE)
diff --git a/goya/common.h b/goya/common.h
index 70b6cc8..0437e15 100644
--- a/goya/common.h
+++ b/goya/common.h
@@ -7,10 +7,10 @@
 
 #include <preboot.h>
 
-#define HL_SPI_NEXT_IMAGE_OFF	0x60000
-#define HL_SPI_MODIFI_IMAGE_OFF	0x60004
-#define HL_SPI_PREBOOT_OFF	0xA0000
-#define HL_SPI_UBOOT_OFF	0xB0000
+#define HL_SPI_NEXT_IMAGE_OFF		0x60000
+#define HL_SPI_MODIFI_IMAGE_OFF		0x60004
+#define HL_SPI_LEGACY_PREBOOT_OFF	0xA0000
+#define HL_SPI_LEGACY_UBOOT_OFF		0xB0000
 
 #define PPBOOT_TEXT_BASE	0x7FF0000000
 #define PREBOOT_TEXT_BASE	(PPBOOT_TEXT_BASE + PPBOOT_SIZE)
diff --git a/goya2/common.h b/goya2/common.h
index 9479f6c..3a65bd5 100644
--- a/goya2/common.h
+++ b/goya2/common.h
@@ -7,10 +7,10 @@
 
 #include <preboot.h>
 
-#define HL_SPI_NEXT_IMAGE_OFF	0x80000
-#define HL_SPI_MODIFI_IMAGE_OFF	0x80004
-#define HL_SPI_PREBOOT_OFF	0xC0000
-#define HL_SPI_UBOOT_OFF	0xD0000
+#define HL_SPI_NEXT_IMAGE_OFF		0x80000
+#define HL_SPI_MODIFI_IMAGE_OFF		0x80004
+#define HL_SPI_LEGACY_PREBOOT_OFF	0xC0000
+#define HL_SPI_LEGACY_UBOOT_OFF		0xD0000
 
 #define PPBOOT_TEXT_BASE		0x7FF0000000
 #define PREBOOT_TEXT_BASE		(PPBOOT_TEXT_BASE + PPBOOT_SIZE)
diff --git a/goya2_arc/common.h b/goya2_arc/common.h
index e05ab43..7165f97 100644
--- a/goya2_arc/common.h
+++ b/goya2_arc/common.h
@@ -7,10 +7,10 @@
 
 #include <preboot.h>
 
-#define HL_SPI_NEXT_IMAGE_OFF	0x80000
-#define HL_SPI_MODIFI_IMAGE_OFF	0x80004
-#define HL_SPI_PREBOOT_OFF	0xC0000
-#define HL_SPI_UBOOT_OFF	0xD0000
+#define HL_SPI_NEXT_IMAGE_OFF		0x80000
+#define HL_SPI_MODIFI_IMAGE_OFF		0x80004
+#define HL_SPI_LEGACY_PREBOOT_OFF	0xC0000
+#define HL_SPI_LEGACY_UBOOT_OFF		0xD0000
 
 #define PCIE_FW_SRAM_BASE	0x1BFF0000
 #define FW_SRAM_BASE		0x1B050000
